locksmith/src/config/config.ts

```diff
@@ -84,6 +84,7 @@ const config = {
   sentry: {
     dsn: 'https://30c5b6884872435f8cbda4978c349af9@o555569.ingest.sentry.io/5685514',
   },
+  lockKeysBucket: 'exports',
```

Comment:
Let's move this _inside_ the `storage` object please. Also, let's rename `bucket` into `uploadsBucket` because that is what this is and let's rename this one `exportsBucket` because it is used for exports.

Reply:
You can remove this line then!